src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -284,7 +284,7 @@ def _alloc_from_dict(self, config_dict):
                     job.private_args.add_from_string(args)
                 except ValueError as err:
                     raise ConfigValidationError(
-                        errors=f"{err}: FORWARD_MODEL {job_name} ({args})\n",
+                        errors=f"{err}: FORWARD_MODEL {job_name!r} ({args!r})\n",
```

Comment:
We end up with ('args') here? This is what we want?

Reply:
Looked at it and, no this is not what we want. It is trying to recreate the `FORWARD_MODEL` keyword line. Changed it.